src/actions/subject.js

```diff
@@ -0,0 +1,110 @@
+import apiClient from 'panoptes-client/lib/api-client'
+import { isNil, length, prepend } from 'ramda'
+import { setState } from '../actions/index'
+import { Image } from 'react-native'
+import getSubjectLocation from '../utils/get-subject-location'
+
+export function fetchUpcomingSubjects(workflowID) {
+  return dispatch => {
+    return new Promise ((resolve, reject) => {
+      apiClient.type('subjects').get({workflow_id: workflowID, sort: 'queued'}).then((subjects) => {
```

Comment:
I'm curious if this `sort: 'queued'` is working. The app queue seems to work fine when I test it in iOS and Android, so it might be fine. However, I don't see in the Panoptes serializer that you can sort subjects by 'queued', and there is also a subject_queue route. Another set of eyes on this would be good.